docs/rest/ReindexFlowReferenceExtension.http

```diff
@@ -0,0 +1,219 @@
+          "code": "US"
+        }
+      ]
+    }
+  ],
+  "code": "network",
+  "base": [
+    "PractitionerRole"
+  ],
+  "type": "reference",
+  "expression": "PractitionerRole.extension.where(url='http://hl7.org/fhir/us/davinci-pdex-plan-net/StructureDefinition/network-reference').value.as(Reference)",
+  "target": [
+    "Organization"
+  ],
+  "multipleOr": true,
+  "multipleAnd": true,
+  "chain": [
+    "name",
+    "address",
+    "partof",
+    "type"
+  ]
+}
+
+###
+# Initiate a reindexing operation.
+# Copy the reindexing job ID into the variable below.
+POST {{baseUrl}}/$reindex HTTP/1.1
+content-type: application/json
+
+{  "resourceType": "Parameters", "parameter": [] }
+
+@reindexId = 438002bc-1995-44d7-aaa5-1f291b7d4a78
+
+
+###
+
+@reindexId = 028575cb-b7bd-4a70-9dea-0787143109da
+
+# Get the status of the reindex job.
+# This should complete quickly because we are only reindexing one resource.
+GET {{baseUrl}}/_operations/reindex/{{reindexId}} HTTP/1.1
+
+###
+# Get the practitioner role with matching network reference info.
+# This should return one value.
+GET {{baseUrl}}/PractitionerRole?network=Organization/AcmeofCTStdNet&_total=accurate HTTP/1.1
+
+###
+# Attempt to get a practitioner role with non-matching network reference info.
+# This shouldn't return any values.
+GET {{baseUrl}}/PractitionerRole?network=Organization/AcmeofDoesNotExist&_total=accurate HTTP/1.1
+
+###
+# Attempt to create a search parameter resource with an expression that is not specific enough.
+# This should fail with the message that the search parameter is unable to be indexed.
+POST {{baseUrl}}/SearchParameter HTTP/1.1
+content-type: {{contentType}}
+
+{
+  "resourceType": "SearchParameter",
+  "id": "practitionerrole-network",
+  "url": "http://hl7.org/fhir/us/davinci-pdex-plan-net/SearchParameter/practitionerrole-network-incomplete-expression",
+  "version": "1.0.0",
+  "name": "Plannet_sp_practitionerrole_network_incomplete_expression",
+  "status": "active",
+  "date": "2018-05-23T00:00:00+00:00",
+  "publisher": "HL7 Financial Management Working Group",
+  "contact": [
+    {
+      "name": "HL7 Financial Management Working Group",
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://www.hl7.org/Special/committees/fm/index.cfm"
+        },
+        {
+          "system": "email",
+          "value": "fm@lists.HL7.org"
+        }
+      ]
+    }
+  ],
+  "description": "Select roles where the practitioner is a member of the specified health insurance provider network",
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166",
+          "code": "US"
+        }
+      ]
+    }
+  ],
+  "code": "network-incomplete-expression",
+  "base": [
+    "PractitionerRole"
+  ],
+  "type": "reference",
+  "expression": "PractitionerRole.extension.where(url='http://hl7.org/fhir/us/davinci-pdex-plan-net/StructureDefinition/network-reference')",
```

Comment:
@Ivanidzo4ka pointed out that this is how the expression is specified in the spec [here](https://build.fhir.org/ig/HL7/davinci-pdex-plan-net/SearchParameter-practitionerrole-network.json.html). Currently looking to see if we can add support to parse expressions that don't specify the exact property in the extension.